testing/flops.h

```diff
@@ -0,0 +1,222 @@
+#ifndef _TESTING_FLOPS_
```

Comment:
You should perhaps the original header for this file, since I presume it was originally from plasma/chameleon...

Reply:
right, this inherits portions from plasma flops.h, but we implement differently and add more to it (can not use the original as is).
we should mention credits in the file comments.